### PR TITLE
Move timestamp to the left

### PR DESCRIPTION
**Pull request by wendigo**
## Description

Before:

Screenshot 2024-10-10 at 22 25 43

After: 

Screenshot 2024-10-10 at 22 23 23

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Result seems fine. I assume you tested it with resizing the window and it behaves ok.

Ideally we would avoid using negative margin values and figure out what is causing the right hand code block to overlap into it or be so close - but given this is our legacy UI I am fine with this fix.

**Comment by martint** [comment]
This seems to have broken in a recent change. It used to look ok before. Do we know what broke it? Can we address the underlying issue instead?

**Comment by mosabua** [comment]
My gut feeling tells me it has to do with the syntax highlighting we added to the list.

**Comment by mosabua** [comment]
Hm .. weirdly I am not observing the spacing issue in https://github.com/trinodb/trino/pull/23748 .. see the screenshots there.

File: core/trino-web-ui/src/main/resources/webapp/assets/trino.css [hunk 1]
```diff
@@ -426,6 +426,7 @@ pre {
 .query-header-timestamp {
     text-align: right;
     color: #b1b1b1;
+    margin-left: -15px;
```
Comment (mosabua) [inline core/trino-web-ui/src/main/resources/webapp/assets/trino.css:429]: I noticed that .query also has the margin-left .. I wonder what happens if we remove that there instead..

**Comment by mosabua** [comment]
Turns out those negative margins (paired with identical value positive padding) are everywhere .. all the way up into the .row setup of bootstrap .. so I think we just ship this since its working. Cleaning up that CSS is probably a waste of time.

**Pull request merged**